public/components/metrics/index.tsx

```diff
@@ -81,7 +82,7 @@ export const Home = ({
     setToasts([...toasts, { id: new Date().toISOString(), title, text, color } as Toast]);
   };
 
-  const onRefreshFilters = (startTime: ShortDate, endTime: ShortDate) => {
+  const onRefreshFilters = (st: ShortDate, et: ShortDate) => {
```

Comment:
it doesn't seem like these are being used - should we remove?

Reply:
good to remove